### PR TITLE
fix: Update the mobile extension name for replaceElementValue API

### PR DESCRIPTION
**Pull request by nicodn**
## Change list

Please provide briefly described change list which are you going to propose. 
 
## Types of changes

What types of changes are you proposing/introducing to Java client?
_Put an `x` in the boxes that apply_

- [ ] No changes in production code.
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Details

The correct command name is `replaceElementValue`

https://github.com/appium/appium-uiautomator2-driver?tab=readme-ov-file#mobile-replaceelementvalue

Currently this method gives an error on Appium 2.4.1:

```
2024-04-23 23:28:35:720 - [AndroidUiautomator2Driver@e813 (57f911b5)] Executing method 'mobile: replaceValue'
2024-04-23 23:28:35:762 - [AndroidUiautomator2Driver@e813 (57f911b5)] Encountered internal error running command: UnknownCommandError: Unknown mobile command "mobile: replaceValue". Only shell, execEmuConsoleCommand, dragGesture, flingGesture, doubleClickGesture, clickGesture, longClickGesture, pinchCloseGesture, pinchOpenGesture, swipeGesture, scrollGesture, scrollBackTo, scroll, viewportScreenshot, viewportRect, deepLink, startLogsBroadcast, stopLogsBroadcast, deviceidle, acceptAlert, dismissAlert, batteryInfo, deviceInfo, getDeviceTime, changePermissions, getPermissions, performEditorAction, startScreenStreaming, stopScreenStreaming, getNotifications, openNotifications, listSms, type, replaceElementValue, pushFile, pullFile, pullFolder, deleteFile, isAppInstalled, queryAppState, activateApp, removeApp, terminateApp, installApp, clearApp, backgroundApp, getCurrentActivity, getCurrentPackage, startActivity, startService, stopService, broadcast, getContexts, getAppStrings, installMultipleApks, lock, unlock, isLocked, refreshGpsCache, startMediaProjectionRecording, isMediaProjectionRecordingRunning, stopMediaProjectionRecording, getConnectivity, setConnectivity, toggleGps, isGpsEnabled, hideKeyboard, isKeyboardShown, pressKey, getDisplayDensity, getSystemBars, fingerprint, sendSms, gsmCall, gsmSignal, gsmVoice, powerAc, powerCapacity, networkSpeed, sensorSet, getPerformanceData, getPerformanceDataTypes, statusBar, screenshots, scheduleAction, getActionHistory, unscheduleAction, getUiMode, setUiMode, sendTrimMemory commands are supported.
```

### CODE REVIEW

**Review by mykola-mokhnach — APPROVED** [review]

**Comment by mykola-mokhnach** [comment]
Thanks @nicodn . Could you please sign the CLA?

**Comment by mykola-mokhnach** [comment]
Substituted with https://github.com/appium/java-client/pull/2171

**Comment by nicodn** [comment]
Thanks... I'm still waiting on legal approval 😴

**Pull request closed without merging**
